docs/usermanual/multinode.rst

```diff
@@ -147,7 +152,7 @@ When the parallel manager, global matrix or global vector are writing to a file,
   rhs.dat.rank.2
   rhs.dat.rank.3
 
-Parallel Manager
+Parallel manager
```

Comment:
.. doxygenfunction:: rocalution::ParallelManager::GetGlobalSize
and
.. doxygenfunction:: rocalution::ParallelManager::GetLocalSize
can't be traced in the doxy file. Need to check the correct syntax.